### PR TITLE
OSCI-3986: process redhat-container-image

### PR DESCRIPTION
**Pull request by Andrei-Stepanov**
Signed-off-by: Andrei Stepanov <astepano@redhat.com>

### CODE REVIEW

**Comment by Andrei-Stepanov** [comment]
PR is ready for merging. It is deployed right now at: https://dashboard.dev.osci.redhat.com/

File: src/components/PageByFilters.tsx [hunk 1]
```diff
@@ -63,6 +63,7 @@ const menuTypes = {
     'Modularity NSVC': 'redhat-module',
     'CentOS Stream NVR': 'koji-build-cs',
     'Fedora NVR': 'koji-build',
+    'RedHat Containers': 'redhat-container-image',
```
Comment (mgrabovsky) [inline src/components/PageByFilters.tsx:66]:
```suggestion
    'Red Hat Containers': 'redhat-container-image',
```
Missing a space.

**Comment by Andrei-Stepanov** [comment]
@mgrabovsky hello, I updated 2 entries:

```
-    tag: string;
+    tag?: string;

-    registry_url: string;
+    registry_url?: string;
```

I would keep task_id as mandatory. I think it is bug in schema. 
It was mixed with `id` that is mandatory in : https://pagure.io/fedora-ci/messages/blob/master/f/schemas/rpm-build.yaml#_15
It was copied, and the author added new `task_id` entry, keeping old `id`.
Container images must have task id in Brew.

I opened issue: https://pagure.io/fedora-ci/messages/issue/148
Task id is unique ID for containers, it is required for this kind of artifacts.

**Comment by Andrei-Stepanov** [comment]
The PR is ready for merge

**Review by mgrabovsky — APPROVED** [review]
OK, looks fine.

**Pull request merged**